deploy/configure.sh

```diff
@@ -72,10 +72,56 @@ if [[ "$AI_MODEL_INFRA_TYPE" == "batch-account" ]]; then
     then
         az batch account login --name ${BATCH_ACCOUNT_NAME} --resource-group ${BATCH_ACCOUNT_RG_NAME}
         # create batch job for custom vision model
-        az batch job create --id ${BATCH_ACCOUNT_POOL_NAME} --pool-id ${BATCH_ACCOUNT_POOL_NAME} --account-name ${BATCH_ACCOUNT_NAME} --account-key ${BATCH_ACCOUNT_KEY}
+        az batch job create --id ${BATCH_ACCOUNT_POOL_NAME} --pool-id ${BATCH_ACCOUNT_POOL_NAME} --account-name ${BATCH_ACCOUNT_NAME} --account-key ${BATCH_ACCOUNT_KEY} || true
```

Comment:
why are we suffixing this true here, will not it mean that we are overriding any errors if there were while creating the job

Reply:
rerunning setup would fails since batch job has already created. We want the script not to stop here.

Reply:
dropped  '|| true' and will address the rerunning and failure issue in the future work